channels_jsonrpc/jsonrpcconsumer.py

```diff
@@ -325,59 +323,12 @@ def notify_channel(cls, reply_channel, method, params):
         reply_channel.send({"text": cls._encode(content)})
 
     @classmethod
-    def __process_notification(cls, data, original_msg):
+    def __process(cls, data, original_msg, is_notification=False):
         """
-                Process an inbound JSON-RPC notificaton
-                :param data: object
-                :return: object
-                """
-
-        if 'method' not in data:
-            logger.warning("The notification doesn't have a 'method'")
```

Comment:
This is not covered anymore, instead of a warning it sends back a Exception with a null "id".
It needs to be modified